### ISSUE TITLE
Selected option from enum does not update UI

### ISSUE THREAD

**Issue by hola-senior** [Textalk/angular-schema-form#458]
I am attempting to run the custom-validators example after modify it to include an enum:

```javascript
$scope.schema = {
     "type": "object",
      "properties": {
           "gender":  {
               "type": "string",
               "enum": ["male","female","other"]
           },
      },
};
$scope.form = ["*"];
```
The resulting page does not update after selecting a gender, but the underlying model does update correctly. I am using Firefox 35. Am I missing something?

**Comment by waleedasif322** [comment]
Can you replicate it in a plunker or jsfiddle?

**Comment by hola-senior** [comment]
I think I've narrowed down how to reproduce. The issue does not appear when using angular 1.4, but does when using 1.2: http://jsfiddle.net/y4hv6w27/2/

**Comment by agzam** [comment]
I can confirm, using ng1.3.16 here. And it's not working.

**Comment by AlexAlbala** [comment]
I am using angular 1.3.14 and this problem has appeared last week for me...

**Comment by mike-marcacci** [comment]
Hey guys! I'm hitting this too, and it's blocking something I need to do today/tomorrow so I'll go ahead and fix it... unless somebody else is already working on it (in which case I'd love to work off of your branch/fork). Anybody have a bit more insight on this?

**Comment by agzam** [comment]
Any updates on the issue guys?

**Comment by davidlgj** [comment]
@mike-marcacci @agzam @AlexAlbala @hola-senior @waleedasif322 
It seems like it was the `trackBy` feature that broke this, PR https://github.com/Textalk/angular-schema-form/pull/418

 I just reverted it and released 0.8.4

**Comment by mike-marcacci** [comment]
Wonderful! Thanks for that @davidlgj – I was going to sit down to figure this one out right now :)
